src/libraries/System.Memory/src/System/Buffers/SequenceReader.cs

```diff
@@ -49,6 +49,14 @@ public SequenceReader(ReadOnlySequence<T> sequence)
         /// </summary>
         public readonly ReadOnlySequence<T> Sequence { get; }
 
+        /// <summary>
+        /// The unread portion of the <see cref="Sequence"/>.
+        /// </summary>
+        public readonly ReadOnlySequence<T> UnreadSequence
+        {
+            get => Sequence.Slice(Position);
```

Comment:
Sorry, one other nit. You don't need the `get =>`.  You can just do `UndreadSequence => Sequence.Slice(Position)`.

Reply:
yep right

Reply:
I saw also below `Position` is on two line with same syntax...it's a formatting nit maybe, let me know it should update that one.